File: pytools/examples/golang/src/server/proto/req_role_rand_name.go
package proto

import (
	"packet"
)

type ReqRoleRandName struct {
}

func ReqRoleRandNameDecode(pack *packet.Packet) *ReqRoleRandName {
	reqRoleRandName := &ReqRoleRandName{}

	return reqRoleRandName
}

File: pytools/data/data.code.lua53.server/ReqRoleRandName.lua
ReqRoleRandName = {}
setmetatable(ReqRoleRandName, {__index = _G})
local _ENV = ReqRoleRandName


function new()
	local tab_obj = {}
	setmetatable(tab_obj, {__index = ReqRoleRandName})
	tab_obj:init()
	return tab_obj
end

function init(self)
end


function decode(self, pack)
	return self
end


